Resources/Prototypes/_NF/Entities/Objects/Storage/pouches.yml

```
@@ -0,0 +1,1323 @@
+    - state: fill-util1-1
+      map: [ "pouchFill1" ]
+      visible: false
+    - state: decal-line-04
+      map: [ "decal2" ]
+      color: "#b33831"
+    - state: decal-religious-cross
+      map: [ "decal1" ]
+      color: "#f6d56a"
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-left
+        color: "#3e3e48"
+      right:
+      - state: inhand-right
+        color: "#3e3e48"
+  - type: Clothing
+    clothingVisuals:
+      belt:
+      - state: equipped-BELT
+        color: "#3e3e48"
+      pocket:
+      - state: equipped-BELT
+        color: "#3e3e48"
+
+- type: entity
+  id: NFPouchMime
+  parent: NFPouchUtility
+  name: mime's pouch
+  components:
+  - type: Sprite
+    layers:
+    - state: base
+      color: "#eeeeee"
+    - state: closed
+      map: ["closeLayer"]
+      color: "#eeeeee"
+    - state: open
+      color: "#eeeeee"
+      map: ["openLayer"]
+      visible: false
+    - state: inside
+      color: "#eeeeee"
+      map: ["insideLayer"]
+      visible: false
+    - state: fill-util4-1
+      map: [ "pouchFill4" ]
+      visible: false
+    - state: fill-util3-1
+      map: [ "pouchFill3" ]
+      visible: false
+    - state: fill-util2-1
+      map: [ "pouchFill2" ]
+      visible: false
+    - state: fill-util1-1
+      map: [ "pouchFill1" ]
+      visible: false
+    - state: decal-mime
+      map: [ "decal1" ]
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-left
+        color: "#eeeeee"
+      right:
+      - state: inhand-right
+        color: "#eeeeee"
+  - type: Clothing
+    clothingVisuals:
+      belt:
+      - state: equipped-BELT
+        color: "#eeeeee"
+      pocket:
+      - state: equipped-BELT
+        color: "#eeeeee"
+
+- type: entity
+  id: NFPouchClown
+  parent: NFPouchUtility
+  name: clown's pouch
+  components:
+  - type: Sprite
+    layers:
+    - state: base
+      color: "#00ff00"
+    - state: closed
+      map: ["closeLayer"]
+      color: "#00ff00"
+    - state: open
+      color: "#00ff00"
+      map: ["openLayer"]
+      visible: false
+    - state: inside
+      color: "#00ff00"
+      map: ["insideLayer"]
+      visible: false
+    - state: fill-util4-1
+      map: [ "pouchFill4" ]
+      visible: false
+    - state: fill-util3-1
+      map: [ "pouchFill3" ]
+      visible: false
+    - state: fill-util2-1
+      map: [ "pouchFill2" ]
+      visible: false
+    - state: fill-util1-1
+      map: [ "pouchFill1" ]
+      visible: false
+    - state: decal-clown
+      map: [ "decal1" ]
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-left
+        color: "#00ff00"
+      right:
+      - state: inhand-right
+        color: "#00ff00"
+  - type: Clothing
+    clothingVisuals:
+      belt:
+      - state: equipped-BELT
+        color: "#00ff00"
+      pocket:
+      - state: equipped-BELT
+        color: "#00ff00"
+
+- type: entity
+  id: NFPouchMailCarrier
+  parent: NFPouchUtility
+  name: mailcarrier's pouch
```

```suggestion
  name: mail carrier's pouch
```
space for consistency